Tests/js/apps/assets/ez-platformuiapp-tests.js

```diff
@@ -314,7 +314,30 @@ YUI.add('ez-platformuiapp-tests', function (Y) {
                 dispatchedEvent,
                 "The `ez:yui-app:ready` event should have been dispatched"
             );
-        }
+        },
+
+        // Regression test for https://jira.ez.no/browse/EZP-27421
+        "Should dispatch the `ez:yui-app:ready` event after plugins are initialized": function () {
+            var app,
+                pluginNS = 'myPlugin',
+                dispatchedEvent = false,
+                Plugin = Y.Base.create('myPlugin', Y.Plugin.Base, [], {
+                    initializer: function () {
+                        this.on('initializedChange', function () {
```

Comment:
the way you test that seems a bit complicated. For readability, I would do the following:
* here, you should set a boolean variable (`pluginInitialized` for instance) to true
* the event listener for `ez:yui-app:ready` should be set right before the app instantiation and in it you assert that the boolean variable is true

no need for the `dispatchedEvent` variable, the event is checked in the previous test.